tests/requirements.txt

```diff
@@ -0,0 +1,6 @@
+docker==7.1.0
+GPUtil==1.4.0
+jinja2==3.1.4
```

Comment:
for what do we need jinja?

Reply:
Same as `transformers`, just for the prompt formatting, but I'll remove those since not needed